notebooks/py_scripts/02_Data_Sync.py

```diff
@@ -65,6 +66,15 @@
 import warnings
 
 warnings.filterwarnings("ignore")
+
+
+env_vars = os.environ
+# check that base dir is defined
+assert (
```

Comment:
I try to rely on other notebooks to explain things that maybe don't belong in the notebook I'm working on. I think I'd just include some markdown here to say: `If you haven't already done so, be sure to set up your Spyglass base directory with [Setup](./00_Setup.ipynb)`